qt-restricted-extras/qscintilla/gen_qsciabstractapis.cpp

```diff
@@ -534,7 +496,9 @@ bool QsciAbstractAPIs_override_virtual_event(void* self, intptr_t slot) {
 }
 
 bool QsciAbstractAPIs_virtualbase_event(void* self, QEvent* event) {
-	return ( (MiqtVirtualQsciAbstractAPIs*)(self) )->virtualbase_event(event);
+
+	return ( (MiqtVirtualQsciAbstractAPIs*)(self) )->MiqtVirtualQsciAbstractAPIs::event(event);
```

Comment:
oops, instead of calling the base class function, this actually calls the override version which goes on to call the language override in a nice infinite recursion ;)

This needs to be `( (MiqtVirtualQsciAbstractAPIs*)(self) )->QsciAbstractAPIs::event(event)` + argument conversion

Reply:
The `c.ClassName` reference was right, but it got caught up in the `fixupProtectedReferences` logic. Probably it will need to be teased apart so that it still builds on Android.